src/python/pants/backend/jvm/tasks/coverage/manager.py

```diff
@@ -83,18 +85,25 @@ def register_junit_options(register, register_jvm_tool):
     # register options for coverage engines
     # TODO(jtrobec): get rid of these calls when engines are dependent subsystems
     Cobertura.register_junit_options(register, register_jvm_tool)
+    Scoverage.register_junit_options(register, register_jvm_tool)
 
   class InvalidCoverageEngine(Exception):
     """Indicates an invalid coverage engine type was selected."""
 
   def get_coverage_engine(self, task, output_dir, all_targets, execute_java):
     options = task.get_options()
-    if options.coverage or options.coverage_processor or options.is_flagged('coverage_open'):
+    processor = options.coverage_processor
```

Comment:
Even if we enable coverage here and set to "scoverage", scoverage can still be turned off via "enable_scoverage". Some consolidation into enabling scoverage to be consistent with cobertura and jacoco here could be done.